commands/call.js

```diff
@@ -19,6 +19,11 @@ module.exports = {
             default: '0',
             alias: 'amount'
         })
+        .option('depositYocto', {
```

Comment:
I don't think that we should add a separate option here.
And we definitely should not use it as a flag.
@mikedotexe I do not understand the motivation in your comment here: https://github.com/near/near-cli/issues/751#issuecomment-882743407

I would say, that we need to design it the next way:
`near call ... ... --deposit 000000001` it would be in N, for backward compatibility + deprecation warning.
`near call ... ... deposit 1N`
`near call ... ... deposit 1yN` for deposit in yoctoNEAR

If we know for sure that `deposit` is always in yoctoNEAR, then let's do it the next way:
`near call ... ... --depositYocto 1` + add deprecation warning for `--deposit`
This one will be less flexible.

+, if we will have `--deposit` and `--depositYocto` at the same time we should not allow to specify them both (there is a conflicting functionality in `yargs`).

@ChaoticTempest please, provide an example of this option usage in the description.

Reply:
In the PR description or somewhere else?

Reply:
Yes, let's put it there :)